cmd/xl-storage-format-v1.go

```diff
@@ -128,12 +128,13 @@ const (
 // ObjectPartInfo Info of each part kept in the multipart metadata
 // file after CompleteMultipartUpload() is called.
 type ObjectPartInfo struct {
-	ETag       string    `json:"etag,omitempty"`
-	Number     int       `json:"number"`
-	Size       int64     `json:"size"`       // Size of the part on the disk.
-	ActualSize int64     `json:"actualSize"` // Original size of the part without compression or encryption bytes.
-	ModTime    time.Time `json:"modTime"`    // Date and time at which the part was uploaded.
-	Index      []byte    `json:"index,omitempty" msg:"index,omitempty"`
+	ETag       string            `json:"etag,omitempty"`
+	Number     int               `json:"number"`
+	Size       int64             `json:"size"`       // Size of the part on the disk.
+	ActualSize int64             `json:"actualSize"` // Original size of the part without compression or encryption bytes.
+	ModTime    time.Time         `json:"modTime"`    // Date and time at which the part was uploaded.
+	Index      []byte            `json:"index,omitempty" msg:"index,omitempty"`
+	CRC        map[string]string `json:"crc,omitempty" msg:"crc,omitempty"`
```

Comment:
did you mean `CSum` ? @klauspost

Reply:
"Cyclic Redundancy Check". Basically the same as checksum. We already use checksums a lot, so tried to make the distinction clearer.

Reply:
I meant if you say

```
fi.CRC["sha256"]
```

v/s

```
fi.Cksum["sha256"] 
```